Equilobe.DailyReport.Models/Jira/JiraIssue.cs

```diff
@@ -234,6 +234,7 @@ public class Worklog
         public string timeSpent { get; set; }
         public int timeSpentSeconds { get; set; }
         public string id { get; set; }
+        public long issueId { get; set; }
```

Comment:
Let's update this whole file and rewrite the properties starting with uppercase letter. Create a separate branch for that and test if it works without specifying the name using DataMember attribute.

Reply:
I tried having this when I was working on bitbucket integration task and without DataMember it didn't seem to work. I will make a task to rewrite the file.